core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1731,6 +1733,8 @@ object SnappySession extends Logging {
         ArrayBuffer[Any]]](CachedPlanHelperExec.BROADCASTS_KEY).getOrElse(
       mutable.Map.empty[BroadcastHashJoinExec, ArrayBuffer[Any]])
 
+    logDebug(s"all bc plans = ${allbroadcastplans} ... size = ${allbroadcastplans.size}")
+
```

Comment:
Will be required later when broadcast plans will also be working with Tokenization.